scripts/leapfrog/pre_redeploy.sh

```diff
@@ -74,6 +88,14 @@ pushd ${LEAPFROG_DIR}
             fi
         popd
         log "variable-migration" "ok"
+        # This message only needs to appears the first time the user variable migration was successful
+        # This way, if a user mistakenly Ctrl-C, the variable migration process will be skipped on
+        # the next run
+        if [[ "${IS_GATING}" != "TRUE" ]]; then
+            notice "Please verify your migrated secrets in /etc/openstack_deploy"
+            warning "DO NOT CTRL-C this process to verify your secrets."
```

Comment:
I don't think this is very helpful, it implies there will be a negative impact from terminating the script however given the use of `.complete` files there don't appear to be any. If someone does it by mistake, this message creates FUD.